ChatSample/ChatSample/Startup.cs

```diff
@@ -30,7 +30,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseSignalR(routes =>
             {
-                routes.MapHub<ChatHub>("chat");
+                routes.MapHub<ChatHub>("/chat");
```

Comment:
Shouldn't be required. Was this throwing? Maybe we fixed that bug in preview2.

Reply:
Yeah, it throws an argument exception if it doesn't start with a /